doc/reference/reference_lua/connpool.rst

```diff
@@ -0,0 +1,188 @@
+
+                           * ``instances`` -- the names of instances to consider as candidates.
+                           * ``replicasets`` -- the names of replica sets whose instances are considered as candidates.
+                           * ``groups`` -- the names of groups whose instances are considered as candidates.
+                           * ``timeout`` -- a connection timeout (in seconds).
+                           * ``buffer`` -- a :ref:`buffer <buffer-module>` used to read a returned value.
+                           * ``on_push`` -- a function to execute when the client receives an out-of-band message. Learn more from :ref:`box_session-push`.
+                           * ``on_push_ctx`` -- an argument of the function executed when the client receives an out-of-band message. Learn more from :ref:`box_session-push`.
+                           * ``is_async`` -- whether to wait for the result of the call.
+
+    :return: a function's return value.
+
+    **Example**
+
+    In the example below, the following conditions are specified to choose an instance to execute the :ref:`vshard.storage.buckets_count <storage_api-buckets_count>` function:
+
+    *   An instance has the ``roles.crud-storage`` role.
+    *   An instance has the ``dc`` label set to ``east``.
+    *   An instance is read-only.
+
+    ..  code-block:: lua
+
+        local connpool = require('experimental.connpool')
+        local buckets_count = connpool.call('vshard.storage.buckets_count',
+                nil,
+                { roles = { 'roles.crud-storage' },
+                  labels = { dc = 'east' },
+                  mode = 'ro' }
+        )
+
+
+..  _connpool_module_connect:
+
+..  function:: connect(instance_name, opts)
+
+    Create a connection to the specified instance.
+
+    :param string instance_name: an instance name.
+    :param table/nil opts: none, any, or all of the following parameters:
+
+                           * ``connect_timeout`` -- a connection timeout (in seconds).
+                           * ``wait_connected`` -- whether to block the connection until it is established:
+
+                               * if ``true`` (default), the connection is blocked until it is established.
+                               * if ``false``, the connection is returned immediately.
+
+                           * ``fetch_schema`` -- whether to fetch schema changes from a remote instance.
+
+    :return: a :ref:`net.box <net_box-module>` connection.
+
+    **Example**
+
+    In the example below, ``connect()`` is used to create the active connection to ``storage-b-002``:
+
+    ..  code-block:: lua
+
+        local connpool = require('experimental.connpool')
+        local conn = connpool.connect("storage-b-002", { fetch_schema = true })
+
+    Once you have a connection, you can execute requests on a remote instance, for example, select data from a space using :ref:`conn.space.\<space-name\>:select() <conn-select>`.
```

Comment:
```suggestion
    Once you have a connection, you can execute requests on the remote instance, for example, select data from a space using :ref:`conn.space.\<space-name\>:select() <conn-select>`.
```

Reply:
Meaning "the instance to which the connection is established". As I understand, within a single `connect()` function call a connection can be established to only 1 instance, so the instance is unique in this context.